README.md

```diff
@@ -54,7 +54,7 @@ freenode.
 This library should always compile with any combination of features on **Rust 1.29**.
 
 Because some dependencies have broken the build in minor/patch releases, to
-compile with 1.22.0 you will need to run the following version-pinning command:
+compile with 1.29.0 you will need to run the following version-pinning command:
 ```
 cargo update -p cc --precise "1.0.41" --verbose
```

Comment:
are we actually still stuck at that version after upgrading to 1.29?

Reply:
I think so. Recently had to [do the same thing for bitcoincore-rpc](https://github.com/rust-bitcoin/rust-bitcoincore-rpc/commit/1b759cbcd463e23f169f5e9ddd526fc0b01688b0#diff-ab015588951d278b386567143c3e8013R5).

Reply:
Yes.